docs/modules/ROOT/pages/how-to/vshn-example/activate-appcat.adoc

```diff
@@ -0,0 +1,81 @@
+= Activate AppCat On {zone}
+
+This page describes how certain AppCat services can be enabled on a {zone}.
+
+. Check that the there is support for the used cloud provider.
+  See https://github.com/vshn/component-appcat/tree/master/packages.
+
+. Set variables
++
+[source,bash]
+----
+export CLUSTER_ID = <cluster>
+export TENANT_ID = <tenant>
+----
+
+. Login to Vault.
++
+[source,bash]
+----
+export VAULT_ADDR=https://vault-prod.syn.vshn.net
+vault login -method=oidc
+----
+
+. Setup API keys for cloud providers
++
+.Cloudscale
+[%collapsible]
+====
+. Login to control.cloudscale.ch.
+. Create a project using the naming scheme `VSHN AppCat objects-${CLUSTER_ID}`, for example `VSHN AppCat objects-lpg-2`.
+. Create an API token in the new project with a name like `appcat-objects-lpg-2` with write access.
+. Store token in Vault
++
+[source,bash]
+----
+parent="clusters/kv/${TENANT_ID}/${CLUSTER_ID}/appcat/objectstorage/provider-cloudscale"
+vault kv put "${parent}" token=<the-cloudscale-token>
+----
+====
++
+.Exoscale
+[%collapsible]
+====
+. Login to portal.exoscale.com
+. Create a new organization using the naming scheme `VSHN AppCat objects-${CLUSTER_ID}`, for example `VSHN AppCat objects-ch-gva-2-0`.
```

Comment:
For exoscale the organization isn't specific for the objects, we we can keep the `objects` part out.

```suggestion
. Create a new organization using the naming scheme `VSHN AppCat ${CLUSTER_ID}`, for example `VSHN AppCat objects-ch-gva-2-0`.
```